cmd/zapi/cmd/get/get.go

```diff
@@ -153,6 +173,55 @@ func parseExpr(spaceID api.SpaceID, expr string) (*api.SearchRequest, error) {
 	}, nil
 }
 
+// parseExprPlusChunk creates an api.WorkerRequest to be used with the client.
+func parseExprPlusChunk(spaceID api.SpaceID, expr string, chunkInfo string) (*api.WorkerRequest, error) {
+	// This is only for testing usng the -chunk flag
+	search, err := zql.ParseProc(expr)
+	if err != nil {
+		return nil, err
+	}
+	proc, err := json.Marshal(search)
+	if err != nil {
+		return nil, err
+	}
+	chunkInfoArr := strings.Split(chunkInfo, "-")
```

Comment:
FYI that I just filed https://github.com/brimsec/zq/issues/1448 : during that work, we can export the data file name parsing function from the archive package & use that single function here, instead of manually needing to parse out the individual chunk fields.

Reply:
Great, I will update this in my next PR.